bindings/dotnetcore/src/dotnetcore.cpp

```diff
@@ -34,13 +34,13 @@ typedef void(*PGatewayDestroyDelegate)(unsigned int moduleIdManaged);
 
 typedef void(*PGatewayStartDelegate)(unsigned int moduleIdManaged);
 
-static PGatewayCreateDelegate GatewayCreateDelegate;
```

Comment:
>static [](start = 0, length = 6)

I assume you removed the "static" for unit testing.  I don't like this, but the only other alternative is to not test this.  Have you tried running 2 dotnet core modules in the same gateway with different delegates?